script/WormholeL2FlexibleVotingDeploy.s.sol

```diff
@@ -115,4 +137,46 @@ contract WormholeL2FlexibleVotingDeploy is Script, ScriptConstants {
     vm.broadcast();
     l1TokenBridge.initialize(address(l2Token));
   }
+
+  function _setupGovernor() internal returns (address, address, bool) {
```

Comment:
maybe a quick comment and/or named returned vars to help make it clear what this function returns?